src/main.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/zairza-cetb/bench-routes/tsdb"
```

Comment:
THis and the line below, both can be kept aside.

```suggestion
import (
	"encoding/json"
	"fmt"
	"net/http"
	"os"
	
	"github.com/zairza-cetb/bench-routes/tsdb"
	"github.com/zairza-cetb/bench-routes/tsdb/v1"
```

Reply:
I think this still needs to be updated? @muskankhedia